docs/source/python_spec.rst

```diff
@@ -65,24 +66,27 @@ struct members, gray text enum values of supported devices and data
 types.*
 
 The ``__dlpack__`` method will produce a ``PyCapsule`` containing a
-``DLManagedTensor``, which will be consumed immediately within
-``from_dlpack`` - therefore it is consumed exactly once, and it will not be
-visible to users of the Python API.
-
-The producer must set the ``PyCapsule`` name to ``"dltensor"`` so that
-it can be inspected by name, and set ``PyCapsule_Destructor`` that calls
-the ``deleter`` of the ``DLManagedTensor`` when the ``"dltensor"``-named
-capsule is no longer needed.
-
-The consumer must transer ownership of the ``DLManangedTensor`` from the
+``DLManagedTensor`` that is compatible with the DLPack and DLPack ABI
+version requested by the consumer. It will be consumed immediately
+within ``from_dlpack`` - therefore it is consumed exactly once, and it
+will not be visible to users of the Python API.
+
+The producer must set the ``PyCapsule`` name to ``"dltensor"`` if ABI
+version 1 is requested and ``"versioned_dltensor"`` if ABI version >= 2
```

Comment:
> I asked in [#98 (review)](https://github.com/dmlc/dlpack/pull/98#pullrequestreview-903401339) and I will ask again: With all these changes flush in, shouldn't we bump the DLPack version at least one more time to include #98 #100 before breaking ABIs? Otherwise, ABI version is never set to 1 in any outside work; it only lives in the dev branch here, and this doc change doesn't really make much sense.

Oh, yes, thanks for noticing! I will propose a PR bumping the DLPack version and rebase here once that's merged.

Reply:
I have bumped the DLPack version here itself since it makes sense to upgrade both the DLPack and ABI version for every ABI break. Sounds good?